src/components/LogView/index.js

```diff
@@ -340,14 +371,19 @@ class LogView extends React.Component {
   render() {
     let j = 0;
     this.indexMap = {};
-
     this.filteredLines = this.props.lines.filter((line, i) => {
       if (!this.props.shouldPrintLine(this.props.bookmarks, line, this.props.filter, this.props.inverseFilter)) {
         return false;
       }
       this.indexMap[i] = j++;
       return true;
     });
+    this.highlightLines = this.filteredLines.filter((line) => {
```

Comment:
Technically, this and filteredLines should be in state. But i'm a scared of running `setState` in `render()`. Let's leave it alone for now.